src/Modules/Frontend/Domain/Block/BlockControllerInterface.php

```diff
@@ -12,4 +12,6 @@ public function __invoke(Request $request, Response $response): string|array;
     public function getResponseOverride(): ?Response;
 
     public static function getModuleBlock(): ModuleBlock;
+
+    public function assign(string $key, mixed $value): void;
```

Comment:
needs to be protected

Reply:
https://github.com/forkcms/forkcms/pull/3555/commits/c24fac573aad3d9e429b9a352c24087b2dd1f862